ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.collection.hbs

```diff
@@ -12,4 +12,4 @@
  **/
  --}}
 <div class="collection"></div>
-<div class="loading"><i class="fa fa-circle-o-notch fa-spin is-critical-animation"></i></div>
```

Comment:
why do we remove the spinner?

Reply:
We're using a [LoadingCompanionView](https://github.com/codice/ddf/pull/3178/files#diff-f77dad5c4415da26f39237f6388a66b2R42) there now instead